smtp/smtp.js

```diff
@@ -185,18 +291,18 @@ class SMTP extends EventEmitter {
 		log(`connecting: ${this.host}:${this.port}`);
 
 		if (this.ssl) {
-			this.sock = tls.connect(
+			this.sock = connect(
 				this.port,
 				this.host,
-				this.ssl,
+				{},
```

Comment:
no boolean param here; supposed to be a hash instead

Reply:
will this impact the README suggestions on how to pass in this parameter?

Reply:
great point -- that's a feature i unknowingly killed. re-added with a typecheck. should probably test for it, then? (unsure how)